terraform/modules/data-share-service/alarms.tf

```
@@ -0,0 +1,179 @@
+      alarm_description = "LEV error rate",
+      error_metric = {
+        name      = "API_RESPONSES.ResponsesFromLev",
+        namespace = local.metric_namespace,
+      },
+      success_metric = {
+        name      = "API_RESPONSES.ErrorsFromLev",
+        namespace = local.metric_namespace,
+      },
+    },
+    prisoner_search_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-prisoner-search-error-rate",
+      alarm_description = "Prisoner search error rate",
+      error_metric = {
+        name      = "API_RESPONSES.ResponsesFromPrisonerSearch",
+        namespace = local.metric_namespace,
+      },
+      success_metric = {
+        name      = "API_RESPONSES.ErrorsFromPrisonerSearch",
+        namespace = local.metric_namespace,
+      },
+    },
+    get_event_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-get-event-error-rate",
+      alarm_description = "Get event error rate",
+      error_metric = {
+        name      = "API_CALLS.GetEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = false
+        }
+      },
+      success_metric = {
+        name      = "API_CALLS.GetEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = true
+        }
+      },
+    },
+    get_events_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-get-events-error-rate",
+      alarm_description = "Get events error rate",
+      error_metric = {
+        name      = "API_CALLS.GetEvents",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = false
+        }
+      },
+      success_metric = {
+        name      = "API_CALLS.GetEvents",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = true
+        }
+      },
+    },
+    delete_event_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-delete-event-error-rate",
+      alarm_description = "Delete event error rate",
+      error_metric = {
+        name      = "API_CALLS.DeleteEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = false
+        }
+      },
+      success_metric = {
+        name      = "API_CALLS.DeleteEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = true
+        }
+      },
+    },
+    publish_event_error_rate = {
+      alarm_name        = "${local.alarm_prefix}-publish-event-error-rate",
+      alarm_description = "Publish event error rate",
+      error_metric = {
+        name      = "API_CALLS.PublishEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = false
+        }
+      },
+      success_metric = {
+        name      = "API_CALLS.PublishEvent",
+        namespace = local.metric_namespace,
+        dimensions = {
+          success = true
+        }
+      },
+    },
+  }
+}
+
+module "error_rate_alarm" {
```

Possibly make this the below as this module will generate many
```suggestion
module "error_rate_alarms" {
```